core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts

```diff
@@ -41,8 +41,25 @@ export class ServiceMetricsComponent implements OnInit, AfterViewInit {
       column: 'View By:',
       label: 'TIME RANGE',
       type: 'select',
-      options: ['Day', 'Week', 'Month', 'Year'],
+      options: [
+        'Last 15 Minutes',
+        'Last 1 Hour',
+        'Last 24 Hours',
+        'Last 7 Days',
+        'Last 30 Days',
```

Comment:
Do we need to add the Last 3 Months option here?

Reply:
Added.